alacritty_terminal/src/ansi.rs

```diff
@@ -1138,20 +1138,20 @@ where
             (b'8', None) => self.handler.restore_cursor_position(),
             (b'=', None) => self.handler.set_keypad_application_mode(),
             (b'>', None) => self.handler.unset_keypad_application_mode(),
-            // String terminator, do nothing (parser handles as string terminator)
+            // String terminator, do nothing (parser handles as string terminator).
             (b'\\', None) => (),
             _ => unhandled!(),
         }
     }
 }
 
 fn attrs_from_sgr_parameters(parameters: &[i64]) -> Vec<Option<Attr>> {
-    // Sometimes a C-style for loop is just what you need
-    let mut i = 0; // C-for initializer
+    // C-for initializer.
```

Comment:
```suggestion
```

Honestly just best to get rid of this probably. Could also rewrite it to a for loop.